### ISSUE TITLE
UpdateAction version 2 not equivalent to UpdateAction version 1

### ISSUE THREAD

**Issue by AlvaroVega** [telefonicaid/perseo-fe#391]
Comes from https://github.com/telefonicaid/perseo-fe/pull/389

https://github.com/telefonicaid/perseo-fe/blob/2f5b2d5a4e75dab80664ee312340a9c6b1327797/lib/models/updateAction.js#L450

        // FIXME: createEntity with upsert true is always updating entity,
        // but is not using params.actionType (append or update)
        // Maybe other methods like v2.updateEntityAttributes are more proper ?

UpdateAction with version 2 was introduced by PR https://github.com/telefonicaid/perseo-fe/pull/305

UpdateAction with filter (and version2) is working well

**Comment by fgalan** [comment]
As commented at Skype, maybe the following could work:

```
 connection.v2.createEntity(changes, { upsert: (actionType !== 'update')}).then( 
```

**Comment by fgalan** [comment]
Fixed by PR #394. But let's have this issue opened for a while... so we can ensure it works in e2e CI.
